### ISSUE TITLE
[FEATURE] Pull Request label "status:blocked"

### ISSUE THREAD

**Issue by eddiejaoude** [EddieHubCommunity/LinkFree#5371]
### Description

Add label to PR `status: blocked` if

- not `chore: profile`
- and there is no `issue` number
- ~~and issue does not have label `status: ready for dev`~~ this can happen in a new PR

Can this be done with Reviewpad?

**Comment by ArslanYM** [comment]
Can you provide the exact description of what we have to change?

**Comment by eddiejaoude** [comment]
What is not clear about the description?

**Comment by ArslanYM** [comment]
@eddiejaoude like what tech we can use to add this feature

**Comment by eddiejaoude** [comment]
Yes, in the description it mentions trying to do it with `Reviewpad` as we already have that available

**Comment by Ananya2001-an** [comment]
Hey @eddiejaoude I can work on this.

**Comment by eddiejaoude** [comment]
Sure @Ananya2001-an 👍

**Comment by Ananya2001-an** [comment]
Hey @eddiejaoude I played around with the code a bit and I had a doubt. You are saying that this label should be added when chore is not profile addition and also should be added if the there is no issue number in the description. But when people create profile we don't have any issue linked so it will get blocked. What do u think?

**Comment by Ananya2001-an** [comment]
Also I guess the first and third condition is achievable, I am not sure about the second one though. But maybe we can use github api to check the labels for that issue.

**Comment by eddiejaoude** [comment]
> when people create profile we don't have any issue linked so it will get blocked

Sorry if not clear, let me try again...

## SKIP
- if the PR has the label `chore: profile`

## ADD
- if the PR there is no issue number

OR

- has issue but the issue does not have label `status: ready for dev`
